caffe2/CMakeLists.txt

```diff
@@ -1178,7 +1178,6 @@ if(USE_XPU)
   if(NOT TARGET torch_xpu_ops)
     message(WARNING "Failed to include ATen XPU implementation target")
   else()
-    target_link_libraries(torch_xpu PRIVATE torch_xpu_ops)
```

Comment:
@cyyever , do you mean the `wholearchive` is able to solve the depency automatically?